README.md

```diff
@@ -62,6 +62,13 @@ and then go to [http://localhost:8000](http://localhost:8000).
 The `asyncapi/studio` image is based on the official `nginx` image.
 Please refer to the [Nginx documentation](https://registry.hub.docker.com/_/nginx/) to learn how to e.g. pass a custom `nginx` configuration or plug in additional volumes.
 
+In some hosting scenarios (e.g. Docker Compose, Kubernetes) the container might not be exposed at the root path of the host.
+Set the environment variable `BASE_URL` to let AsyncAPI Studio know from where to resolve static assets:
```

Comment:
I guess that you tested it on your environment and everything works? I tested it locally and base path `/` works and also preview works so for me it's most important to not broke https://studio.asyncapi.com. In your case with custom `BASE_URL` does it work like should? I don't have a way to test this on e.g. kubernetes so I rely on you here :)

Reply:
Yes, my main goal was also to not break "normal" usage 😀

I tested `BASE_URL` locally, yes!